src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+    {
+      person.age && (
+      <p className="Person__age">{`I am ${person.age}`}</p>
+      )
+    }
+    {
+      person.isMarried !== true ? (
```

Comment:
there is no need to write `... !== true`

you can just write `person.isMarried` , this will automatically be converted to a boolean value, and if its `true` then the next part of code will work